packages/perseus/src/widgets/interactive-graphs/mafs-graph.test.tsx

```diff
@@ -0,0 +1,93 @@
+import {screen, render, waitFor} from "@testing-library/react";
+import {userEvent as userEventLib} from "@testing-library/user-event";
+import React from "react";
+
+import {ApiOptions} from "../../perseus-api";
+
+import {MafsGraph} from "./mafs-graph";
+
+import type {InteractiveGraphProps} from "./types";
+import type {PerseusInteractiveGraphWidgetOptions} from "../../perseus-types";
+import type {UserEvent} from "@testing-library/user-event";
+
+function getBaseMafsGraphProps(): InteractiveGraphProps {
+    const mafsGraphBaseProps: PerseusInteractiveGraphWidgetOptions = {
+        step: [1, 1],
+        gridStep: [1, 1],
+        snapStep: [1, 1],
+        markings: "graph",
+        labels: [],
+        showProtractor: false,
+        showRuler: false,
+        rulerLabel: "",
+        range: [
+            [-10, 10],
+            [-10, 10],
+        ],
+        graph: {type: "segment"},
+        correct: {type: "segment"},
+        rulerTicks: 4,
+    };
+
+    const widgetBaseProps: InteractiveGraphProps = {
+        ...mafsGraphBaseProps,
+        alignment: null,
+        apiOptions: {
+            ...ApiOptions.defaults,
+        },
+        containerSizeClass: "small",
+        findWidgets: () => [],
+        isLastUsedWidget: false,
+        onBlur: () => {},
+        onChange: () => {},
+        onFocus: () => {},
+        problemNum: 1,
+        reviewModeRubric: {
+            ...mafsGraphBaseProps,
+        },
+        static: true,
+        trackInteraction: () => {},
+        widgetId: "mafs-graph",
+        linterContext: {} as any,
+    };
+
+    return widgetBaseProps;
+}
+
+describe("MafsGraph", () => {
+    let userEvent: UserEvent;
+    beforeEach(() => {
+        userEvent = userEventLib.setup({
+            advanceTimers: jest.advanceTimersByTime,
+        });
+    });
+
+    it("renders", () => {
+        render(<MafsGraph {...getBaseMafsGraphProps()} box={[400, 400]} />);
+
+        expect(screen.getByTestId("mafs-graph__wrapper")).toBeInTheDocument();
```

Comment:
We don't use this test ID anywhere else, so it feels like this test isn't adding much value. If this test fails, what will that tell us? Likely just that the test ID has changed or gone missing.

I'd rather have this first test verify the assumptions made by the second test. E.g. it could check for the presence of a `circle.movable-point-hitbox`.